common/utils/dates.ts

```diff
@@ -1,5 +1,6 @@
 import { DateTypes, london } from './format-date';
 import { DateRange } from '../model/date-range';
+import { isSameDay } from 'date-fns';
```

Comment:
fwiw if you see bundle size issues, you can import functions directly like `import isSameDay from 'date-fns/isSameDay'`, but really Webpack should handle the modules/tree-shaking for you correctly here

Reply:
I've had a poke around locally and I think it is tree-shaking correctly – if I swap them for the specific modules, I don't see a noticeable change in bundle size. 😢